test/passing/doc_comments.mli

```diff
@@ -351,3 +351,82 @@ end
     [ A lot of    spaces ]
 
     [Very looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong] *)
+
+(** {[
+  for i = 1 to 3
+  do
+    Printf.printf "let x%d = %d\n" i i
+  done
+]} *)
+
+(** {[
+  print_newline ();
+  List.iter
+    (fun s -> Printf.printf "let ( %s ) = Pervasives.( %s )\n" s s)
+    ["+"; "-"; "*"; "/"]
+]}  *)
+
+(** {[
+  #use "import.cinaps";;
+
+  List.iter all_fields ~f:(fun (name, type_) -> printf "\nexternal get_%s
+  : unit -> %s = \"get_%s\"" name type_ name)
+]} *)
+
+(** {[
+
+
+  List.iter all_fields ~f:(fun (name, type_) -> printf "\nexternal get_%s
+  : unit -> %s = \"get_%s\"" name type_ name)
+]} *)
+
+(** {[
+  let x = 1 in
+
+  (* fooooooo *)
+  let y = 2 in
+  (* foooooooo *)
+  z
+]} *)
+
+(** {[
+   let this  = is_short
+]}
+
+{[
+  does not parse: verbatim
++/+/+ /+/+/ +/+//+/+/+/+/+/+/+/
++/+/+ /+/+/ +/+//+/+/+/+/+/+/+/
++/+/+ /+/+/ +/+//+/+/+/+/+/+/+/
++/+/+ /+/+/ +/+//+/+
+]}
+
+{[
+[@@@ocamlformat "break-separators = after"]
+
+let fooooooooooooooooo =
+[ foooooooooooooooooooooooooooooooo
+; foooooooooooooooooooooooooooooooo
+; foooooooooooooooooooooooooooooooo ]
+
+]}
+
+{[
+let fooooooooooooooooo =
+[ foooooooooooooooooooooooooooooooo
+; foooooooooooooooooooooooooooooooo
+; foooooooooooooooooooooooooooooooo ]
+
+]} *)
+
+
+
+(**
+  This is a comment with code inside
+  {[
+    (** This is a comment with code inside
+        {[ let code inside = f inside ]}
```

Comment:
I've added this example of nested comments. It seems that odoc is not happy about it.

Reply:
https://github.com/ocaml/odoc/issues/375